source/core/types/ut_assert_result.tpb

```diff
@@ -7,6 +7,7 @@ create or replace type body ut_assert_result is
     self.object_type   := 0;
     self.result        := a_result;
     self.error_message := a_error_message;
+    self.caller_info   := ut_assert_processor.who_called_expectation();
```

Comment:
for performance reasons, we could limit those calls to only occur when assert failed.
Also, maybe we should rename the type to be `ut_expectation_result`?